style.css

```diff
@@ -1,100 +1,376 @@
+}
+
+.my-logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.container-headline {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+  background-image: url("images/header.svg");
+  background-repeat: repeat;
+}
+
+.headline {
+  display: flex;
+  height: 300px;
+  padding: 50px 24px;
+  flex-wrap: wrap;
+}
+
+.primary-text {
+  display: flex;
+  align-items: flex-end;
+  justify-content: center;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  color: #172b4d;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  align-items: flex-start;
+}
+
+.social-media {
+  height: 25%;
+  width: 50%;
+  align-items: flex-start;
+}
+
+.media-icons {
+  margin-top: 12px;
+  list-style: none;
+  display: flex;
+  padding: 0;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+#media-text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #7f8cff;
+}
+
+.icon-list {
+  display: inline-flex;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  margin-top: 100px;
+  align-items: center;
+  justify-content: space-between;
+  background: #e5e5e5;
+}
+
+.tonic-1 {
+  display: grid;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  border-radius: 16px;
+  grid-template-columns: repeat(6, 16.66%);
+  grid-template-rows: 1fr 60px 50px 1fr 60px 1fr;
+  margin-top: 75px;
+}
+
+.stories-1 {
+  display: grid;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  border-radius: 16px;
+  grid-template-columns: repeat(6, 16.66%);
+  grid-template-rows: 1fr 100px 30px 1fr 40px 1fr;
+  margin-top: 75px;
+}
+
+.tonic-2 {
+  display: grid;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  border-radius: 16px;
+  grid-template-columns: repeat(6, 16.66%);
+  grid-template-rows: 1fr 60px 50px 1fr 60px 1fr;
+  margin-top: 75px;
+}
+
+.stories-2 {
+  display: grid;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  border-radius: 16px;
+  grid-template-columns: repeat(6, 16.66%);
+  grid-template-rows: 1fr 100px 30px 1fr 40px 1fr;
+  margin-top: 75px;
+  margin-bottom: 75px;
+}
+
+.photo-grid-1 {
+  grid-column: 1 / 7;
+  grid-row: 1 / 2;
+  justify-self: center;
+  margin-top: 16px;
+}
+
+.project-title {
+  grid-column: 1 / 7;
+  grid-row: 2 / 3;
+  justify-self: start;
+  margin-left: 16px;
+  margin-bottom: 0;
+  margin-top: 12px;
+  margin-right: 30px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.canopy-segment {
+  grid-column: 1 / 7;
+  grid-row: 3 / 4;
+  display: flex;
+  align-items: center;
+}

```

Comment:
- From your layout, you can see you're repeating yourself and your code is not dry which is not a good practice.
- It will be very easy if you used flexbox on all your card and not mixing it up with grid and flex as technically for your case flex will be a great deal for you to use to lay out all the items on your card. Please let's improve our styling here.